tobot_tooling/src/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+set(TOBOT_TOOLING_SOURCE_FILES
+"logger.cpp"
```

Comment:
Is tobot_asserts included somewhere else when debug config is selected?

Reply:
No forgot to add it 🥴